parlai/crowdsourcing/tasks/pairwise_per_turn_eval/task_config/model_opts.yaml

```diff
@@ -0,0 +1,48 @@
+blender_90M: >
+    --model-file zoo:blender/blender_90M/model
+    --beam-block-full-context False
+    --beam-block-ngram 3
+    --beam-context-block-ngram 3
+    --beam-min-length 20
+    --beam-size 10
+    --inference beam
+
+blender_90M_beam_min_length_0: >
+    --model-file zoo:blender/blender_90M/model
+    --beam-block-full-context False
+    --beam-block-ngram 3
+    --beam-context-block-ngram 3
+    --beam-min-length 0
+    --beam-size 10
+    --inference beam
+# TODO: this is just for testing - remove this model opt entry before open-sourcing
```

Comment:
Yeah I'd remove this block because we didn't test this case in the paper and so it might just confuse people

Reply:
Whoops - good catch!